repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/RateLimitingHandler.java

```diff
@@ -42,16 +42,22 @@
 import java.io.ByteArrayOutputStream;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import static javax.servlet.http.HttpServletResponse.*;
 
 /* Responsible for handling requests and responses to rate limiting, also tracks and provides limits */
 public class RateLimitingHandler {
 
     private static final Logger LOG = org.slf4j.LoggerFactory.getLogger(RateLimitingHandler.class);
+    private static final Set<MimeType> SUPPORTED_MIME_TYPES = Stream.of(
```

Comment:
That is totally an option, but I'm not super eager to add a Guava dependency just for that. Unless someone particularly wants that solution. 

After looking through this code again, I'd really love to rewrite the rate limiting filter in Scala. Eventually, the service, too.